.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Don't think we want these files to be checked in. Add `.vscode/` to `.gitignore` instead. We could consider adding a `.editorconfig`, but even then ;)